iib/common/tracing.py

```diff
@@ -0,0 +1,211 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""Configures the Global Tracer Provider and exports the traces to the OpenTelemetry Collector.
+
+The OpenTelemetry Collector is configured to receive traces via OTLP over HTTP.
+The OTLP exporter is configured to use the environment variables
+OTEL_EXPORTER_OTLP_TRACES_ENDPOINT and OTEL_EXPORTER_OTLP_TRACES_HEADERS to
+configure the endpoint and headers for the OTLP exporter.
+The OTLP* environment variables are configured in the docker-compose.yaml
```

Comment:
docker-compose and podman-compose are for the local testing environment and you already added instructions for running the local env. So, I would suggest removing this comment from here. 

For the other envs (QE, Stage, Prod) we are running, these variables would be passed using ansible. It's the task [CLOUDDST-18418] @yashvardhannanavati is working on!